README.md

```diff
@@ -27,11 +27,11 @@ Repository to describe, develop, document and test the DeviceStatus API family
 * Note: Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until a new release is created. For example, changes may be reverted before a release is created. **For best results, use the latest available public release**.
 
 <!-- following part left in for now but could be omitted going forward -->
-* The latest available and released version 0.5.1 of the API is available within the [release-0.5.1 branch](https://github.com/camaraproject/DeviceStatus/tree/release-0.5.1)
+* The latest available release version 0.6 (with API version 0.6.0-alpha.1) is available within the [release-0.6 branch](https://github.com/camaraproject/DeviceStatus/tree/release-0.6)
```

Comment:
If you keep the part, please link not any more to the "release branches", as they were never useful and can be omitted, but to the release tag. With that the link would look like (after you have created the "r0.6" release:

```suggestion
* The latest available release version 0.6 (with API version 0.6.0-alpha.1) is available [here](https://github.com/camaraproject/DeviceStatus/tree/r0.6)
```